apps/src/gamelab/AnimationTab/NewListItem.jsx

```diff
@@ -52,10 +56,17 @@ var NewListItem = React.createClass({
   },
 
   render: function () {
+    var dynamicStyles = {
+      dottedBorder: {
+        backgroundColor: Radium.getState(this.state, 'main', ':hover')
+            ? color.lightest_purple : undefined
+      }
+    };
```

Comment:
The way that I'd have written this (open discussion as to which is best):
```
var staticStyles = {
  ...
  dottedBorder: {
    ..
  },
  dottedBorderHovered: {
    backgroundColor: color.lightest_purple
  }
};

...
var dottedStyle = [staticStyles.dottedBorder, Radium.getState(this.state, 'main', ':hover') && staticStyles.dottedBorderHovered];
```

Thoughts?